python/shotgun_model/shotgun_model.py

```diff
@@ -384,7 +384,7 @@ def reset(self):
 
     def _load_data(
         self, entity_type, filters, hierarchy, fields, order=None, seed=None, limit=None,
-        columns=None, additional_filter_presets=None
+        columns=None, editable_columns=None, additional_filter_presets=None
```

Comment:
I don't think `additional_filter_presets` are public yet right? They're not documented anyway. I've moved the argument to the end to be safe. :)